app/controllers/DraftMetadataUploadController.scala

```diff
@@ -35,4 +41,30 @@ class DraftMetadataUploadController @Inject() (
       }
     }
   }
+
+  def saveDraftMetadata(consignmentId: java.util.UUID): Action[MultipartFormData[TemporaryFile]] = secureAction.async(parse.multipartFormData) {
+
+    implicit request: Request[MultipartFormData[TemporaryFile]] =>
+      val successPage = routes.DraftMetadataChecksController.draftMetadataChecksPage(consignmentId)
+      val userId = request.asInstanceOf[Request[AnyContent]].token.userId
+
+      val uploadBucket = s"tdr-draft-metadata-${applicationConfig.frontEndInfo.stage}"
+      val uploadKey = s"$userId/$consignmentId/draft-metadata.csv"
```

Comment:
For now the key should not include the user id, ie should be: `$consignmentId/draft-metadata.csv`